lib/bot/get-price.js

```diff
@@ -4,7 +4,38 @@ const { ALERT_TYPES, MAX_PAGES, CHROME_DEBUG } = require('../constants.js');
 const createBrowser = require('../browser.js');
 const urlUtil = require('url');
 
-async function getPriceForFlight ({ from, to, date, number, isPointsBooking, passengerCount, alertType, nonStopOnly }, browser, lock) {
+const waitTillHTMLRendered = async (page, timeout = 30000) => {
```

Comment:
Does this basically wait until the size of the rendered HTML stops loading? Neat trick.

Reply:
Yes, in case some scripts are modifying the content on the page still. I liked the concept of this code in addition to waiting for networkidle. Found that code when I was researching waitForSelector so I can't really take credit for it.